airflow/config_templates/default_airflow.cfg

```diff
@@ -170,6 +170,9 @@ web_server_host = 0.0.0.0
 # The port on which to run the web server
 web_server_port = 8080
 
+# Root URL to use for the web server
+web_server_url_prefix: /
```

Comment:
keep the defaults empty.. also it should be `=` insted of `:`?

Reply:
It actually works with both : and =, but I put = to keep coherency